doc/source/namespaces.rst

```diff
@@ -13,40 +13,40 @@ first connect to the cluster.
 
    ray.init(namespace="hello")
    # or using ray client
-   ray.client().namespace("world").connect()
+   ctx = ray.client().namespace("world").connect()
 
 Named actors are only accessible within their namespaces.
 
 .. code-block:: python
 
-    import
+    import ray
 
     @ray.remote
     class Actor:
       pass
 
     # Job 1 creates two actors, "orange" and "purple" in the "colors" namespace.
-    ray.client().namespace("colors").connect()
+    ctx = ray.client().namespace("colors").connect()
```

Comment:
Let's use the `with` syntax here and below (recommended), we can leave the context in one of them and say this is  the way to do it if more explicit control over the lifetime of the connection is needed (and let's call the context `context` instead of `ctx`).

Reply:
Ahh sounds good!